dockstore-client/pom.xml

```diff
@@ -296,8 +298,33 @@
             <groupId>jakarta.annotation</groupId>
             <artifactId>jakarta.annotation-api</artifactId>
         </dependency>
-    </dependencies>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-validator</artifactId>
+            <version>6.1.7.Final</version>
```

Comment:
on a more micro-level, if you did need to add these dependencies, I think you can skip the versions. Our dockstore-bom should cover some of these at least

Reply:
I removed all the version numbers.